### PR TITLE
Update gather data from company emails rule

### PR DESCRIPTION
**Pull request by synnoveisabel**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ Feedback from @danielmackay and @bradystroud 

> 2. What was changed?

✏️ Updated the content of the rule to better highlight the benefits.
Added related link to the "checked by xxx" rule and vice a versa.

> 3. Did you do pair or mob programming (list names)?

✏️

### CODE REVIEW

File: rules/generate-insightful-reports-via-company-emails/rule.md [hunk 1]
```diff
@@ -7,16 +7,22 @@ authors:
...
 
 <!--endintro-->
 
-Emails should follow a set of [email rules](/rules-to-better-email) that make communication consistent and structured. When emails have standard format and content, data analysis is possible because the content can be parsed.  
+1. Gain insights into a company's operations
+2. Detect bottlenecks
+3. Better distribute workload
+4. Make strategic decisions
+
+**Microsoft Graph API** is a great tool to extract Outlook data. The extracted data is not just collected; it is transformed and displayed in Power BI reports by utilizing **Open AI** to analyze the emails. AI is able to analyze the data without having to worry about spelling, casing, nicknames or slang.
```
Comment (JackDevAU) [inline rules/generate-insightful-reports-via-company-emails/rule.md:23]:
```suggestion
**Microsoft Graph API** is a great tool to extract Outlook data. The extracted data is not just collected; it is transformed and displayed in Power BI reports by utilizing **Open AI** to analyze the emails. AI is able to analyze the data without having to worry about spelling, case sensitivity, nicknames or slang. 
```

> casing

I don't think this is a word in this context.

File: rules/generate-insightful-reports-via-company-emails/rule.md [hunk 2]
```diff
@@ -7,16 +7,22 @@ authors:
     url: https://www.ssw.com.au/people/isabel-sandstroem/
 created: 2023-12-15T01:24:13.247Z
 guid: 9477fb4c-b872-4c82-af96-34d1cabd1d7c
-
+related:
+  - checked-by-xxx 
 ---
 
-The emails exchanged within your company are more than just communication tools; they are a rich source of data. By analyzing this data, you can gain insights into a company's operations.
+The emails exchanged within your company are more than just communication tools; they are a rich source of data. By analyzing this data, you can .
```
Comment (bradystroud) [inline rules/generate-insightful-reports-via-company-emails/rule.md:14]: finish this sentence

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: rules/generate-insightful-reports-via-company-emails/rule.md (hunk 2 above)
Comment (synnoveisabel) [reply rules/generate-insightful-reports-via-company-emails/rule.md:14]: The numbered list completes the sentence (the dot is a typo), but if it is unclear I'll change it.

image
**Figure: The complete text in the rule**

File: rules/generate-insightful-reports-via-company-emails/rule.md [hunk 3]
```diff
@@ -7,16 +7,22 @@ authors:
     url: https://www.ssw.com.au/people/isabel-sandstroem/
 created: 2023-12-15T01:24:13.247Z
 guid: 9477fb4c-b872-4c82-af96-34d1cabd1d7c
-
+related:
+  - checked-by-xxx 
 ---
 
-The emails exchanged within your company are more than just communication tools; they are a rich source of data. By analyzing this data, you can gain insights into a company's operations.
+The emails exchanged within your company are more than just communication tools; they are a rich source of data. By analyzing this data, you can
```
Comment (JackDevAU) [inline rules/generate-insightful-reports-via-company-emails/rule.md:14]:
```suggestion
The emails exchanged within your company are more than just communication tools; they are a rich source of data. By analyzing this data, you can:
```

**Review by bradystroud — APPROVED** [review]

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**